src/common/components/DataTable/Paginator.tsx

```diff
@@ -63,12 +66,14 @@ export const Paginator: FC<Props> = ({
               onChange={option => onPageSizeChange(Number(option.value))}
             />
           </div>
-          <span className='text-muted'>items per page</span>
+          <span className='text-muted'>{t('itemsPerPage', { ns: 'common' })}</span>
         </Col>
         {/* Display the range of results currently showing */}
         <Col className='d-flex justify-content-end align-items-center'>
           <span className='text-muted' style={{ marginRight: '10px' }}>
-            {count !== 0 ? `${rangeStart} - ${rangeEnd} of ${count}` : 'No Results'}
+            <Trans ns='common' i18nKey="paginationRange" count={count}>
+              Showing {{ start: rangeStart }} - {{ end: rangeEnd }} of {{ count }}
```

Comment:
This text within the component is not actually used, but left here to help provide context when viewing the component. (The text is replaced with what is in the translation files)